hedera-node/hapi-utils/src/main/java/com/hedera/node/app/hapi/utils/fee/FeeBuilder.java

```diff
@@ -440,7 +440,7 @@ public static int getStateProofSize(final ResponseType responseType) {
                 : 0;
     }
 
-    protected long calculateRBS(final TransactionBody txBody) {
+    protected static long calculateRBS(final TransactionBody txBody) {
```

Comment:
Not just this method, but probably all of the public or protected methods of this class should be static.

Reply:
Done. Thanks